qualitas/templates/dashboards/cnx_repos.html

```diff
@@ -93,58 +93,152 @@
   </script>
 </head>
 
-<body data-feedly-mini="yes">
+<body data-feedly-mini="yes"><!DOCTYPE html>
+<head>
+  <title>CNX Repos Dashboard</title>
+  <meta charset="UTF-8">
+  <meta name=viewport content="width=700">
+  <link rel="shortcut icon" href="newspaper.ico" type="image/x-icon">
+  <style>
+  body {
+    font-family: Noto Sans, Arial, sans-serif;
+    font-size: small;
+  }
+
+  table { width: 700px; }
+
+  .noborder { border:0; }
+
+  td {
+   border: 1px solid gray;
+   text-align: left;
+   vertical-align: top;
+   padding: 7px;
+   width: 50%;
+  }
+  </style>
+
+  <script>
+
+  function getLatestCNXTag(repo) {
+    tags_url = 'https://api.github.com/repos/' + repo + '/tags';
```

Comment:
Be careful with assigning variables before they are defined. One thing I like to use is `"use strict:"` which will give you a warning if you reference a variable before assignment. 

In the latest Javascript they have altered the way you assign variables a bit. The `var` assignment is no longer the recommended method. The two that should be considered before `var` are `let` and `const`.

- `let`: a variable that is likely to change
- `const`: a variable that is meant to be immutable (something you don't want to change)
- `var`: to be used as a last resort